cm_service_restart.py

```diff
@@ -36,6 +36,19 @@ def select_from_items(item_type, items):
         print 'Bad input received...'
         return select_from_items(item_type, items)
 
+
+# Rolling restart method (using few defaults - Role types, role names. It will take all)
+def rolling_restart(service_rr):
+    print service_rr
+    batch_size = raw_input("Number of worker roles to restart together in a batch")
+    fail_count_threshold = raw_input("Stop rolling restart if more than this number of worker batches fail")
+
+    options = ["true", "false"]    # options for stale config
+    stale_config_only = select_from_items('Restart roles on nodes with stale config? Default is false', options)   # Needs to be a user selection
+
+    # rr_command = service_rr.rolling_restart(batch_size, fail_count_threshold, None, stale_config_only, None, None, None).wait()
+    # return rr_command
```

Comment:
Commented for now since I need to confirm which service can be given a shot before executing this. 
Here I am getting few inputs from the user to set the parameters as per their wish. The ones I have passed as None will be defaulted to whatever they are.
Still a test needs to be done on this. But once this piece is merged, dont wanna take risk to lose any code . lol